docs/MYSQL_WC.md

```diff
@@ -0,0 +1,52 @@
+MySQL Working Copy
+-----------------------
+
+In order to use a [MySQL](https://www.mysql.com/) working copy, you need to have server running MySQL. MySQL 8.0 and later is officially supported by Kart (MySQL 5.6 and later are largely compatible but not officially supported).
+
+### MySQL partitioning
+
+MySQL databases are designed so that they can be used for multiple tasks simultaneously without those tasks interfering with each other. This is achieved by storing data from different dasks in different databases.
```

Comment:
```suggestion
MySQL servers are designed so that they can be used for multiple apps simultaneously without those apps interfering with each other. This is usually achieved by storing data from different apps in different databases.
```